virttest/qemu_vm.py

```diff
@@ -1484,6 +1484,22 @@ def sort_key(dev):
         if params.get("defconfig", "yes") == "no":
             devices.insert(StrDev('nodefconfig', cmdline=" -nodefconfig"))
 
+        # Add device intel-iommu, it must be added before any virtio devices
+        if (params.get('intel_iommu') == 'yes' and
+                devices.has_device('intel-iommu')):
+            iommu_params = {
+                'intremap': params.get('iommu_intremap', 'on'),
+                'device-iotlb': params.get('iommu_device-iotlb', 'on'),
```

Comment:
No other comments, only change `iommu_device-iotlb` to `iommu_device_iotlb` to align other parameters.

Reply:
@PaulYuuu Ah yes, updated. Thanks.